coda/coda_validate/views.py

```diff
@@ -1,30 +1,21 @@
-import datetime
+import json
 import random
-try:
-    # the json module was included in the stdlib in python 2.6
-    # http://docs.python.org/library/json.html
-    import json
-except ImportError:
-    # simplejson 2.0.9 is available for python 2.4+
-    # http://pypi.python.org/pypi/simplejson/2.0.9
-    # simplejson 1.7.3 is available for python 2.3+
-    # http://pypi.python.org/pypi/simplejson/1.7.3
-    import simplejson as json
-
-from django.shortcuts import render_to_response
-from django.template import RequestContext
-from django.core.paginator import Paginator
+
+import datetime
```

Comment:
yeah, i didn't spend any time scrutinizing the output. 

>Some of the codalib imports are split across two groups. I'm thinking rope did this, right?

Not sure what you mean here though.